CMake/FindLua.cmake

```diff
@@ -57,30 +56,30 @@ FIND_LIBRARY (LUA_LIBRARY NAMES ${LUA_LIBRARY_NAME}
   /Library/Frameworks
 )
 
-IF (NOT LUA_INCLUDE_DIR)
-    MESSAGE(FATAL_ERROR "Could Not Find Lua Include Dir")
-ENDIF (NOT LUA_INCLUDE_DIR)
+if(NOT LUA_INCLUDE_DIR)
+  message(FATAL_ERROR "Could Not Find Lua Include Dir")
+endif()
 
-IF (NOT LUA_LIBRARY)
-    MESSAGE(FATAL_ERROR "Could Not Find Lua Library")
-ENDIF (NOT LUA_LIBRARY)
+if(NOT LUA_LIBRARY)
+  message(FATAL_ERROR "Could Not Find Lua Library")
+endif()
 
 
-IF (LUA_INCLUDE_DIR AND LUA_LIBRARY)
-    SET (Lua_FOUND TRUE)
-ENDIF (LUA_INCLUDE_DIR AND LUA_LIBRARY)
+if(LUA_INCLUDE_DIR AND LUA_LIBRARY)
+  set(Lua_FOUND TRUE)
+endif()
 
-IF (Lua_FOUND)
-    IF (NOT Lua_FIND_QUIETLY)
-        MESSAGE(STATUS "Found ${LUA_INTERPRETER_TYPE} library: ${LUA_LIBRARY}")
-    ENDIF (NOT Lua_FIND_QUIETLY)
-ELSE (Lua_FOUND)
-   IF (Lua_FIND_REQUIRED)
-       MESSAGE(FATAL_ERROR "Could not find ${LUA_INTERPRETER_TYPE}")
-   ENDIF (Lua_FIND_REQUIRED)
-ENDIF (Lua_FOUND)
+if(Lua_FOUND)
+  if(NOT Lua_FIND_QUIETLY)
+    message(STATUS "Found ${LUA_INTERPRETER_TYPE} library: ${LUA_LIBRARY}")
+  endif()
+else (Lua_FOUND)
+ if(Lua_FIND_REQUIRED)
+   MESSAGE(FATAL_ERROR "Could not find ${LUA_INTERPRETER_TYPE}")
```

Comment:
And we don't need to check `Lua_found` or the else condition at this point, we know it had to be set here.